### PR TITLE
chore: [FX-4058] update jira deployment

### PR DESCRIPTION
**Pull request by dmaklygin**
[FX-4058]

### Description

We should adjust jira-deployment GH Action to trigger Jenkins job to create JIRA deployment properly.

### How to test

- Use alpha package or branch name to test

### Development checks

- [x] Add changeset according to [guidelines](https://github.com/toptal/davinci-github-actions/blob/master/_docs/contribution/changeset-guidelines.md) (if needed)

<details>
<summary>PR commands</summary>
<br />

List of available commands:

- `@toptal-anvil ping reviewers` - Ping FX team for review

</details>

[FX-4058]: https://toptal-core.atlassian.net/browse/FX-4058?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by dmaklygin** [comment]
@toptal-anvil ping reviewers

File: create-jira-deployment/README.md [hunk 1]
```diff
@@ -24,7 +24,14 @@ Not specified
...
 
-Not specified
+All ENV Variables, defined in a GH Workflow are also passed to a GH Action. It means, the might be reused as is.
+This is a list of ENV Variables that are used in GH Action:
+
+| name                  | description                                                                      |
+| --------------------- | -------------------------------------------------------------------------------- |
+| `JENKINS_USER`        | Jenkins user                                                                     |
+| `JENKINS_BUILD_TOKEN` | Jenkins build token. Keep in mind that tokens for `temploy` and `staging` differ |
+| `PROXY`               | Proxy to use for Jenkins                                                         |
```
Comment (sashuk) [inline create-jira-deployment/README.md:34]: Just wondering, why do we need a proxy here, the `jenkins_url` is not enough?

**Review by sashuk — APPROVED** [review]
Looks good (taking into account https://github.com/toptal/davinci/pull/2077)!

File: create-jira-deployment/README.md (hunk 1 above)
Comment (dmaklygin) [reply create-jira-deployment/README.md:34]: it's a Proxy URL, includes username and password

Comment (sashuk) [reply create-jira-deployment/README.md:34]: @dmaklygin Do you mean for Jenkins to use so Jenkins can get to Jira?

Comment (dmaklygin) [reply create-jira-deployment/README.md:34]: Nope, to get to Jenkins server

Comment (augustobmoura) [reply create-jira-deployment/README.md:34]: Probabaly to connect GHA to Jenkins, right?

**Review by augustobmoura — APPROVED** [review]
I don't see anything wrong, couldn't test it manually, but everything works as expected

File: create-jira-deployment/README.md (hunk 1 above)
Comment (sashuk) [reply create-jira-deployment/README.md:34]: Great, thanks!

**Pull request merged**